### ISSUE TITLE
questions about comparison method

### ISSUE THREAD

**Issue by yuequnwang** [lugia-xiao/TCGN#4]
Thank you for the excellent work you have provided. Could you please share the model parameters for the HistToGene method mentioned in your paper? Alternatively, could you provide the training hyperparameters? The results I obtained from running the HisTogene method are significantly worse than those reported in your paper. I have used the parameters as shown in the attached diagram. I look forward to your reply!
![OF)MI36{$5_CQPEM_(QLAVX](https://github.com/lugia-xiao/TCGN/assets/116630142/8fea04a8-98d1-4111-ba1f-010b45e977f8)

**Comment by lugia-xiao** [comment]
It is quite surprising that your repeat of HistToGene is lower than that of my repeat. Unfortunately, my part of work did not contains calculating the result of previous methods so I did not have the trained model in my computer, but I think the hyperparameter settings are:
HisToGene(
    n_genes= xxx, 
    patch_size=112, 
    n_layers=4, 
    dim=1024, 
    learning_rate=1e-5, 
    dropout=0.1, 
    n_pos=64
)
This may produce a similar results as in the original HistToGene paper. If you have any other questions, feel free to reach out to me.
